core/build.gradle

```diff
@@ -1,6 +1,7 @@
-sourceCompatibility = 1.7
+sourceCompatibility = 1.8
```

Comment:
This results in that we can't run the program in desktop (atleast not until we change something else)

Reply:
It changes the language level from 7 to 8, allowing, among other things, the usage of lambda expressions. Did you click on the 'sync now' link that popped up for the gradle? That might fix the other issue you had as well

Reply:
Yes, I did. But running on andrioid worked fine, so we dont need to change it. You just asked to be informed if things didnt work on our computers

Reply:
Think I fixed it, please let me know if you still have problems with it